src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.v2.js

```diff
@@ -181,7 +181,8 @@ export function transformFormToVAOSAppointment(state) {
       desiredDate: `${data.preferredDate}T00:00:00+00:00`,
     },
     locationId: data.vaFacility,
+    reasonCode: getReasonCode(data),
```

Comment:
Type of care is missing in the VA appointment details page. 
Add `serviceType: typeOfCare.idV2,` to pass in type of care

Reply:
I think we need to create another ticket to update all mock data to match the current swagger docs.